api.js

```diff
@@ -175,29 +177,32 @@ module.exports = function (u, p, d, s, o) {
                 method: call[0],
                 params: call[1]
             };
-            if(!json.params){
+            if (!json.params) {
                 json.params = {};
             }
             return json;
         });
 
-        call("ExecuteMultiCall", { calls: formattedCalls }, callback);
+        call("ExecuteMultiCall", {
+            calls: formattedCalls
+        }, callback);
     };
 
     var authenticate = function (callback) {
         var params = {
             userName: userName,
             password: password,
+            sessionId: sessionId,
```

Comment:
Why session id here?

Reply:
Good catch, that was part of me trying something else that didn't work and I should have removed it.  I'll update.